[delivers - #171442590] Componente Tilemap

AdriAlvarez00 opened the pull request:
Explicacion en src/Documentation.md
Si queréis verlo bonito que la ventana tenga una aspect ratio multiplo de 32:32

AdriAlvarez00 committed:
Basic tilemap and tbeginning tileset

AdriAlvarez00 committed:
.lib para Debug y Release

A 3 LNK Error de dejar la carrera

AdriAlvarez00 committed:
Cursed Tilemap

Funciona? Pero empareja mal las texturas. Importante exportar en CSV, nada de base64

AdriAlvarez00 committed:
Tileset spacing

AdriAlvarez00 committed:
Merge branch 'Tilemap-Tileson' into 171442590-Tilemap

AdriAlvarez00 committed:
Escalado y carga automática de tilesets

AdriAlvarez00 committed:
Tilemap Component

AdriAlvarez00 committed:
Merge remote-tracking branch 'origin/master' into 171442590-Tilemap

AndresRuizB commented on src/SourceFiles/TileMap.h:
```diff
@@ -0,0 +1,24 @@
+	int mapCols_, mapRows_;
+	tson::Map tMap_;
+	std::vector<tson::Tileset> tileSets_;
+	SDLTexturesManager* tm;
```
Barrabaja

AndresRuizB reviewed (changes requested):
pon la barrabaja xd

AdriAlvarez00 committed:
draw() es const y barrabaja™

EstebenR reviewed (changes requested):
Men borraste el box2d.lib xd

AdriAlvarez00 committed:
.lib extra

los .lib ya están en src/Box2D/lib/Debug y src/Box2D/lib/Release, que he cambiado la configuracion. Dejo este en la misma ruta de antes por si hay algún problema

JuanmaSalva reviewed (approved):
Oka

EstebenR commented:
Resulta que faltaban las librerias debug y release de box2d

EstebenR reviewed (changes requested):
Las librerias debug y release no pueden ir en carpetas llamadas 'debug' o 'release' porque están en el gitignore poner otro nombre asi sea test y prod

AdriAlvarez00 committed:
Debug_lib y Release_lib

.gitignore ya no ignora las nuevas librerías

AdriAlvarez00 committed:
Merge branch 'master' into 171442590-Tilemap

mmur07 reviewed (approved):
Mola un montón 👍

EstebenR committed:
Movidas inclusiones innecesarias en un .h

AdriAlvarez00 commented:
Resueltos los merge conflicts
